weave/integrations/google_ai_studio/cassettes/google_ai_studio_test/test_content_generation.yaml

```diff
@@ -0,0 +1,2356 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate, zstd
+      Connection:
+      - keep-alive
+      User-Agent:
+      - python-requests/2.32.3
+    method: GET
+    uri: https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json
```

Comment:
please remove this litellm request from this and every other cassette

Reply:
Removed the litellm request and responses from the cassettes.
However, it seems like the cassettes are missing the request and responses for the respective ai studio testcases. Any suggestion as to what could be wrong here?

Reply:
maybe too much was removed?

Reply:
i wonder if your machine cached the results resulting in no calls to record

Reply:
I tried on a fresh codespace, still the cassettes are empty other than the litellm ones. This is how I'm generating the cassettes:
```shell
pytest --record-model=rewrite weave/integrations/google_ai_studio/google_ai_studio_test.py::test_content_generation
```